gatsby/pages.rst

```diff
@@ -1,7 +1,15 @@
 Pages
 =====
 
-Pages can be considered similar to HTML documents in a website, but here, internally they are React components instead with way more functionality and can even be dynamically generated.
+The core part of a website are pages.
+
+Every site has at least one HTML page (for example a single page application or a landing page).
+
+GatsbyJS is a static site generator, so it has the concept of pages itself.
+
+The only difference is that they are not standard HTML documents, but internally they are React components that will be converted into static HTML at build time.
```

Comment:
Reading this carefully, I think that only lines 8 and 10 can be 1 paragraph, instead of 2, by removing the blank line that separates them.

The other sentences in this introduction are separate concepts, so keeping them as separate paragraphs makes sense.